src/plugins/data-card-tool/components/case-attribute.tsx

```diff
@@ -100,24 +112,49 @@ export const CaseAttribute: React.FC<IProps> = observer(props => {
     setEditFacet("");
   };
 
-  const activateInput = (facet: EditFacet) => {
+  const activateInput = (facet: EditFacet, editing: boolean) => {
     setEditFacet(facet);
-    if (facet === "name"){
+    if (facet === "name" && !editing){
       setLabelCandidate(getLabel());
     }
-    if (facet === "value"){
+    if (facet === "value" && !editing){
       setValueCandidate(getValue());
     }
     setCurrEditAttrId(attrKey);
   };
 
+  function deleteAttribute(){
+    console.log("? deleteAttribute: ", attrKey);
+    if(attrKey){
+      content.dataSet.removeAttribute(attrKey);
+    }
+  }
+
+  const AlertContent = () => {
+    return (
+      <p>
+        Are you sure you want to remove the <em style={{ fontWeight: "bold"}}>{ getLabel() }</em> attribute from the Data Card?
+        If you remove it from this card it will delete the data in the field,
+        and it will also be removed from all the Cards in this collection.
+      </p>
+    );
+  };
+
+  const [showAlert] = useCautionAlert({
+    title: "Delete Attribute",
+    content: AlertContent,
+    confirmLabel: "Delete Attribute",
+    onConfirm: () => deleteAttribute()
+  });
+
   const handleDeleteAttribute = () => {
-    // confirmation dialog
+    showAlert();
   };
 
   const pairClassNames = classNames(
     `attribute-name-value-pair ${attrKey}`,
-    {"editing": editFacet === "name" || "value"}
+    {"editing": editFacet === "name" || "value"},
```

Comment:
This will set `editing` to either `true` or `"value"`, so I believe it will always be included as a `className`.

Reply:
Oh, thanks.  Fixed!